jmclient/jmclient/yieldgenerator.py

```diff
@@ -172,9 +177,14 @@ def oid_to_order(self, offer, amount):
         if not filtered_mix_balance:
             return None, None, None
         jlog.debug('mix depths that have enough = ' + str(filtered_mix_balance))
-        mixdepth = self.select_input_mixdepth(filtered_mix_balance, offer, amount)
-        if mixdepth is None:
+
+        try:
+            mixdepth, utxos = self._get_order_inputs(
+                filtered_mix_balance, offer, required_amount)
+        except NoIoauthInputException:
+            jlog.error('unable to fill order, no suitable IOAUTH UTXO found')
```

Comment:
This, of course, is technically correct, but I'm wondering, could we make this error message more user friendly?

Reply:
We could add something like "In order to spend UTXOs from a mixdepth there needs to be at least one normal UTXO.". Not sure if the meaning of "normal" is clear enough. Would you add anything else?

Reply:
maybe replace "normal UTXO" with "standard wallet UTXO".

Reply:
Maybe something like - "Unable to fill order, no suitable IOAUTH UTXO found. In order to spend coins (UTXO's) from a mixdepth using coinjoin, there needs to be at least one standard wallet UTXO (not timelocked fidelity bond or other non-standard)."

Reply:
I agree with this language @kristapsk (except the apostrophe in "UTXO's" - apostrophes are for elision and possession, not plurals - unfortunately 80% of native speakers don't even understand this any more :) ).

Obviously you're choosing the verbose option and I think that's the right choice here.

Reply:
Oh, sorry about bad English, not a native speaker, as you know. Another example of people trying to enforce rules of your own native language to English.

Reply:
> not a native speaker, as you know

Not criticizing here, that's what I'm saying, lots of native speakers get it wrong :smile:

Reply:
done in 7ccdaeb